Solutions/Corvus.Monitoring.ApplicationInsights.Specs/AiOperationsInstrumentationSpecs.cs

```diff
@@ -118,20 +118,22 @@ public void WhenOperationWithPostStartPropertiesAndMetricsAddedSeparatelyFinishe
         [Test]
         public void WhenChildOperationFinishesTelemetryIncludesParentId()
         {
-            string testActivityId = Activity.Current.Id;
-            string parentOpActivityId;
-            string childOpActivityId;
+            string? testActivityId = Activity.Current?.Id;
```

Comment:
Since the test basically requires there to be an activity id to work, there's a bug in the test if this is null, so I don't think this should be declared as `string?`. I think that in this case, the right way to nullable-ref-typify this would be:

```cs
string testActivityId = Activity.Current.Id ?? throw new InvalidOperationException("Test expects a current Activity");
```